mkosi/installer/apt.py

```diff
@@ -40,6 +40,15 @@ def __str__(self) -> str:
 
 
 class Apt(PackageManager):
+    DOCUMENTATION_GLOBS = [
+        "usr/share/doc/*",
+        "usr/share/doc/*/copyright",
```

Comment:
That's already included in the line above, isn't it?

Reply:
Ah shit this was an include to keep those